Library/Formula/gdal.rb

```diff
@@ -308,3 +323,4 @@ def caveats
     system "#{bin}/ogrinfo", "--formats"
   end
 end
+
```

Comment:
There already is one in this file, the GitHub UI and most text editors don't show it.